e2e/infrastructure/MosaicHttp.spec.ts

```diff
@@ -54,7 +65,7 @@ describe('MosaicHttp', () => {
         after(() => {
             return listener.close();
         });
-        it('standalone', (done) => {
+        it('Announce MosaicDefinitionTransaction', (done) => {
```

Comment:
How is this related to `MosaicHttp` ? Shouldn't it be located in `TransactionHttp` ?

Reply:
This is to setup the test mosaic before calling the mosaicHttp. So that the test is not rely on static mosaicId. As the Id changes per node / network